account_invoice_open_sale/readme/USAGE.rst

```diff
@@ -0,0 +1,19 @@
+============================
+Open sale order from invoice
+============================
+This module shows a button to navigate to a customer invoice's related sale
+order if there is one.
```

Comment:
Move this to DESCRIPTION, and replace it here with a user oriented description (probably very similar, just with other pronouns).